homeassistant/components/miele/strings.json

```diff
@@ -257,6 +257,9 @@
           "plate_step_warm": "Warming"
         }
       },
+      "power_disc_level": {
+        "name": "PowerDisc level"
```

Comment:
Commercial name is PowerDisk: https://www.miele.de/product/11841220/powerdisk-all-in-1-vorratsset-6-stueck-gs-setd6

I don't know why API is using "c" instead of "k".